.github/workflows/beta.yaml

```diff
@@ -88,11 +107,121 @@ jobs:
       - name: Push Docker image
         uses: docker/build-push-action@v4
         with:
+          builder: ${{ steps.buildx.outputs.name }}
           build-args: |
-            CCACHE_REMOTE_STORAGE=${{ secrets.PORLA_CCACHE_REMOTE_STORAGE_WRITE }}
             GITVERSION_SEMVER=${{ steps.gitversion.outputs.fullSemVer }}
           context: .
-          push: true
+          # Skip pushing when PR from a fork
+          push: ${{ !github.event.pull_request.head.repo.fork }}
```

Comment:
I think we can keep `push: true` in the beta workflow - it only runs on main, never on pull requests.